typescript/infra/config/environments/mainnet3/owners.ts

```diff
@@ -80,6 +80,9 @@ export const ethereumChainOwners: ChainMap<OwnableConfig> = Object.fromEntries(
           validatorAnnounce: DEPLOYER, // unused
           testRecipient: DEPLOYER,
           fallbackRoutingHook: DEPLOYER,
+          // Ensure we know keep the Safe and ICA addresses for each chain somewhere in the config
```

Comment:
nit: slight grammar mistake but could you also describe the problem for why we needed this